CRM/Mailing/DAO/Mailing.php

```diff
@@ -1195,6 +1222,84 @@ public static function &fields() {
           ],
           'add' => '3.3',
         ],
+        'start_date' => [
+          'name' => 'start_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Mailing Start Date'),
+          'description' => ts('When the mailing started to go out'),
+          'required' => FALSE,
+          'usage' => [
+            'import' => FALSE,
+            'export' => FALSE,
+            'duplicate_matching' => FALSE,
+            'token' => FALSE,
+          ],
+          'where' => 'civicrm_mailing.start_date',
+          'default' => NULL,
+          'table_name' => 'civicrm_mailing',
+          'entity' => 'Mailing',
+          'bao' => 'CRM_Mailing_BAO_Mailing',
+          'localizable' => 0,
+          'html' => [
+            'type' => 'Select Date',
+            'formatType' => 'activityDateTime',
+          ],
+          'readonly' => TRUE,
+          'add' => NULL,
+        ],
+        'end_date' => [
+          'name' => 'end_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Mailing End Date'),
+          'description' => ts('When the mailing finished going out.'),
+          'required' => FALSE,
+          'usage' => [
+            'import' => FALSE,
+            'export' => FALSE,
+            'duplicate_matching' => FALSE,
+            'token' => FALSE,
+          ],
+          'where' => 'civicrm_mailing.end_date',
+          'default' => NULL,
+          'table_name' => 'civicrm_mailing',
+          'entity' => 'Mailing',
+          'bao' => 'CRM_Mailing_BAO_Mailing',
+          'localizable' => 0,
+          'html' => [
+            'type' => 'Select Date',
+            'formatType' => 'activityDateTime',
+          ],
+          'readonly' => TRUE,
+          'add' => NULL,
+        ],
+        'status' => [
+          'name' => 'status',
+          'type' => CRM_Utils_Type::T_STRING,
+          'title' => ts('Mailing Status'),
+          'description' => ts('The status of this mailing'),
+          'maxlength' => 12,
+          'size' => CRM_Utils_Type::TWELVE,
+          'usage' => [
+            'import' => FALSE,
+            'export' => FALSE,
+            'duplicate_matching' => FALSE,
+            'token' => FALSE,
+          ],
+          'where' => 'civicrm_mailing.status',
+          'default' => 'Scheduled',
+          'table_name' => 'civicrm_mailing',
+          'entity' => 'Mailing',
+          'bao' => 'CRM_Mailing_BAO_Mailing',
+          'localizable' => 0,
+          'html' => [
+            'type' => 'Select',
+          ],
+          'pseudoconstant' => [
+            'callback' => 'CRM_Core_SelectValues::getMailingJobStatus',
```

Comment:
I think we'll need a new pseudoconstant function to include the Draft status.

Reply:
Or just add "Draft" to the existing function? Probably wouldn't hurt anything 🤷🏻